portfolio/src/main/webapp/script.js

```diff
@@ -296,38 +296,75 @@ function buildSectionEntryDescription(description) {
 }
 
 /**
- * Processes and appends a list of greetings.
+ * Processes and appends a list of comments. Renders a message
+ * if there are no comments to show.
  * 
- * @param {!Array<string>} greetings A list of greetings.
+ * @param {!Array<string>} comments A list of comments.
  */
-function renderGreetings(greetings) {
+function renderComments(comments) {
   const commentList = document.querySelector('.comments-list');
-  const greetingFragments = greetings.map(buildGreetingFragment);
 
-  commentList.append(...greetingFragments);
+  if (comments.length) {
```

Comment:
use early returns to avoid having to use `else`, which add complexity to the code and makes it harder to read.

```suggestion
  if (!comments.length) {
    commentList.append(buildNoCommentsFragment());
    return;
  }
  
  const commentFragments = comments.map(buildCommentFragment);
  commentList.append(...commentFragments);
  
```